workbench-core/base/src/services/metadataService.ts

```diff
@@ -3,8 +3,25 @@
  *  SPDX-License-Identifier: Apache-2.0
  */
 
+import { QueryCommandOutput } from '@aws-sdk/client-dynamodb';
+import { ZodTypeAny } from 'zod';
+import AwsService from '../aws/awsService';
+import QueryParams from '../interfaces/queryParams';
+import {
+  addPaginationToken,
+  DEFAULT_API_PAGE_SIZE,
+  getPaginationToken,
+  MAX_API_PAGE_SIZE
+} from '../utilities/paginationHelper';
+import { validateAndParse } from '../utilities/validatorHelper';
+
 export class MetadataService {
-  public constructor() {}
+  private readonly _awsService: AwsService;
+
+  public constructor(awsService: AwsService) {
+    this._awsService = awsService;
```

Comment:
We can narrow the scope down by just passing in `ddbService` instead of `awsService`